Pods/EPContactsPicker.swift

```diff
@@ -153,19 +178,7 @@ open class EPContactsPicker: UITableViewController, UISearchResultsUpdating, UIS
         switch CNContactStore.authorizationStatus(for: CNEntityType.contacts) {
             case CNAuthorizationStatus.denied, CNAuthorizationStatus.restricted:
                 //User has denied the current app to access the contacts.
-                
-                let productName = Bundle.main.infoDictionary!["CFBundleName"]!
-                
-                let alert = UIAlertController(title: "Unable to access contacts", message: "\(productName) does not have access to contacts. Kindly enable it in privacy settings ", preferredStyle: UIAlertControllerStyle.alert)
-                let okAction = UIAlertAction(title: "Ok", style: UIAlertActionStyle.default, handler: {  action in
-                    completion([], error)
-                    self.dismiss(animated: true, completion: {
-                        self.contactDelegate?.epContactPicker(self, didContactFetchFailed: error)
-                    })
-                })
-                alert.addAction(okAction)
-                self.present(alert, animated: true, completion: nil)
-            
+                self.contactDelegate?.presentAlert()
```

Comment:
I think the name of this method is too generic, maybe presentContactEditScreen?

Reply:
this is not the "add contact" method, this is presenting an alert when user did not give permissions for contacts

Reply:
maybe I can change it to presentPermissionsAlert?